### PR TITLE
Test: unit test for pkg/misc/other.go

### PR DESCRIPTION
**Pull request by tanryberdi**
ref: https://github.com/cectc/dbpack/issues/68

### Ⅰ. Describe what this PR did
Added unit-test for `pkg/misc/other.go`

### Ⅱ. Does this pull request fix one issue?
yes

### CODE REVIEW

File: pkg/misc/other_test.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	for name, c := range cases {
+		t.Run(name, func(t *testing.T) {
+			got := IsFloat64Equal(c.a, c.b)
+			if got != c.want {
+				t.Errorf("IsFloat64Equal(%v, %v) = %v, want %v", c.a, c.b, got, c.want)
+			}
+		})
+	}
+}
+func TestIsFloat32Equal(t *testing.T) {
```
Comment (dk-lockdown) [inline pkg/misc/other_test.go:47]: please add a blank line before this func

Comment (tanryberdi) [reply pkg/misc/other_test.go:47]: done

**Review by dk-lockdown — COMMENTED** [review]
LGTM

**Comment by adofsauron** [comment]
LGTM

**Comment by fatelei** [comment]
LGTM

**Comment by fatelei** [comment]
LGTM

**Pull request merged**